packages/modules/rbac/src/models/rbac-role-policy.ts

```diff
@@ -0,0 +1,28 @@
+import { model } from "@medusajs/framework/utils"
+import RbacPolicy from "./rbac-policy"
+import RbacRole from "./rbac-role"
+
+const RbacRolePolicy = model
+  .define("rbac_role_policy", {
+    id: model.id({ prefix: "rlpl" }).primaryKey(),
+    role: model.belongsTo(() => RbacRole),
+    scope: model.belongsTo(() => RbacPolicy),
```

Comment:
suggestion: wouldn't it make more sense to call this `policy`?
```suggestion
    policy: model.belongsTo(() => RbacPolicy),
```

Reply:
resurfacing this one